### ISSUE TITLE
Let's restore the CI (again)

### ISSUE THREAD

**Issue by bmagyar** [ros-controls/ros2_controllers#119]
@destogl @v-lopez I managed to update things to the right version so that linters don't fail anymore but we have the boost::shared_ptr pluginlib issue again even though I have the flags set... Have they changed in foxy?

**Comment by v-lopez** [comment]
I'm not aware of the issue. What is it about?

**Comment by bmagyar** [comment]
Note the non-CI changes here: https://github.com/ros-controls/ros2_controllers/pull/104

The error reported by the CI is 
```
  --- stderr: joint_state_controller
  In file included from /root/upstream_ws/install/controller_manager/include/controller_manager/controller_manager.hpp:37,
                   from /root/target_ws/src/ros2_controllers/joint_state_controller/test/test_load_joint_state_controller.cpp:18:
  /opt/ros/foxy/include/pluginlib/class_loader.hpp:55:10: fatal error: boost/shared_ptr.hpp: No such file or directory
     55 | #include <boost/shared_ptr.hpp>
        |          ^~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  make[2]: *** [CMakeFiles/test_load_joint_state_controller.dir/build.make:63: CMakeFiles/test_load_joint_state_controller.dir/test/test_load_joint_state_controller.cpp.o] Error 1
  make[1]: *** [CMakeFiles/Makefile2:155: CMakeFiles/test_load_joint_state_controller.dir/all] Error 2
  make: *** [Makefile:141: all] Error 2
  ---
```

I'm pretty sure the fix is something trivial but it's a needle in a haystack for me ATM :D

**Comment by gavanderhoorn** [comment]
Is `boost` listed as a build dependency?

**Comment by bmagyar** [comment]
We don't have boost and have explicit flags for pluginlib to not try using boost.

**Comment by gavanderhoorn** [comment]
Yeah, I figured. It was a stupid comment to post.

**Comment by destogl** [comment]
You have to have the variable set also in all packages which use pluginlib (directly or indirectly).

This means all controller will probably have to have this...

**Comment by bmagyar** [comment]
Resolved by #122
